cmd/sql/flow.go

```diff
@@ -256,6 +266,7 @@ func runCommand() *cobra.Command {
 	}
 	cmd.SetHelpFunc(executeHelp)
 	cmd.Flags().StringVarP(&projectDir, "project-dir", "p", ".", "")
+	cmd.Flags().BoolVarP(&generateTasks, "generate-tasks", "g", false, "")
```

Comment:
we should look to add a description of what an individual flag means for the command and also a Long/Short command description for all these commands. Would be quite useful for users to get started quickly

Reference: https://github.com/astronomer/astro-cli/blob/main/cmd/airflow.go#L120-L134